git-branchless-revset/src/pattern.rs

```diff
@@ -135,6 +135,7 @@ pub(super) fn make_pattern_matcher_set(
     ctx: &mut Context,
     repo: &Repo,
     matcher: Box<dyn PatternMatcher>,
+    commits_to_match: Option<CommitSet>,
```

Comment:
suggestion: We should rename the fields/variables `visible_commits` to `commits_to_match` in `MatcherNameSetQuery` and the rest of this function.

Reply:
Sure thing. Done!